Userland/Applications/FileManager/DirectoryView.cpp

```diff
@@ -519,14 +519,18 @@ void DirectoryView::do_delete(bool should_confirm)
     delete_paths(paths, should_confirm, window());
 }
 
+bool DirectoryView::can_modify()
```

Comment:
`can_modify` was a decent name when it was a local bool, but I think as a member function on the view it should be a bit more descriptive. maybe, `can_modify_current_selection()` ?

Reply:
Yep, good suggestion. I've updated the method name.
Thank you ;)